tensorboard/webapp/metrics/metrics_module.ts

```diff
@@ -77,6 +77,8 @@ export function alertActionProvider() {
     ),
     EffectsModule.forFeature([MetricsEffects]),
     AlertActionModule.registerAlertActions(alertActionProvider),
+    AppRoutingModule,
```

Comment:
My intuition is that it is safer to import these modules before creating your MetricsEffects. We've seen at least one subtle bug related to the order in which effects are registered (#4952). By importing the dependent modules, first, you allow them to assume that they will be the first to handle any actions that they fire.